### PR TITLE
fix: selectable accessable radio inputs

### PR DESCRIPTION
**Pull request by jswhisperer**
fix: selectable accessable radio inputs
two fold issue event wasnt propagating but also input was too small, linked the label to input with for > to id attributes

### CODE REVIEW

**Comment by jswhisperer** [comment]
Some reading material https://github.com/Semantic-Org/Semantic-UI-React/pull/1517

**Review by chrisvxd — COMMENTED** [review]
Unfortunately I don't think this is the right fix. I'm closing in favour of #733.

File: packages/core/components/AutoField/fields/RadioField/index.tsx [hunk 1]
```diff
@@ -28,19 +28,36 @@ export const RadioField = ({
...
+              id={`${label}-${name}-${index}-${id}`}
               type="radio"
               className={getClassName("radioInput")}
               value={option.value as string | number}
+<<<<<<< Updated upstream
               name={name}
               onChange={({ target: { value } }) =>
                 onChange(safeJsonParse(value) || value)
               }
+=======
```
[the remark this answers is not in the document]
Comment (jswhisperer) [reply packages/core/components/AutoField/fields/RadioField/index.tsx:47]: sorrrrry :(

File: packages/core/components/AutoField/fields/RadioField/index.tsx [hunk 2]
```diff
@@ -28,19 +28,36 @@ export const RadioField = ({
       el="div"
     >
       <div className={getClassName("radioGroupItems")} id={id}>
-        {field.options.map((option) => (
+        {field.options.map((option, index) => (
           <label
+            htmlFor={`${label}-${name}-${index}-${id}`}
```
[the remark this answers is not in the document]
Comment (jswhisperer) [reply packages/core/components/AutoField/fields/RadioField/index.tsx:33]: Agreed,  I had thought that it resolved the issue but I'm realising next dev server and hot-reloading has been the bane of my issues. I know I need to kill the server every change now.

File: packages/core/components/AutoField/fields/ArrayField/index.tsx [hunk 3]
```diff
@@ -174,9 +174,6 @@ export const ArrayField = ({
                   e.stopPropagation();
                   setHovering(false);
                 }}
-                onClick={(e) => {
-                  e.preventDefault();
-                }}
```
[the remark this answers is not in the document]
Comment (jswhisperer) [reply packages/core/components/AutoField/fields/ArrayField/index.tsx:179]: Oh nice one, yeah came to the same conclusion nice one

**Pull request closed without merging**